Discussion/Discussion/Presentation/Comments/Base/ParentCommentView.swift

```diff
@@ -108,29 +108,29 @@ public struct ParentCommentView: View {
                 }, label: {
                     comments.voted
                     ? (CoreAssets.voted.swiftUIImage.renderingMode(.template))
-                    : (CoreAssets.vote.swiftUIImage.renderingMode(.template))
+                    : CoreAssets.vote.swiftUIImage.renderingMode(.template)
+                    
                     Text("\(comments.votesCount)")
-                        .foregroundColor(Theme.Colors.textPrimary)
                     Text(DiscussionLocalization.votesCount(comments.votesCount))
                         .font(Theme.Fonts.labelLarge)
-                        .foregroundColor(Theme.Colors.textPrimary)
+                    
                 }).foregroundColor(comments.voted
-                                   ? Theme.Colors.accentXColor
+                                   ? Theme.Colors.accentColor
                                    : Theme.Colors.textSecondaryLight)
                 Spacer()
                 Button(action: {
                     onReportTap()
                 }, label: {
                     comments.abuseFlagged
-                    ? CoreAssets.reported.swiftUIImage
-                    : CoreAssets.report.swiftUIImage
+                    ? (CoreAssets.reported.swiftUIImage.renderingMode(.template))
```

Comment:
do we need outer brackets here?

Reply:
https://github.com/edx/edx-mobile-marketplace-ios/pull/90#discussion_r1811951525